src/main/java/com/clanki/commands/ReviewCommand.java

```diff
@@ -0,0 +1,49 @@
+package com.clanki.commands;
+
+import com.clanki.objects.Flashcard;
+import com.clanki.objects.FlashcardList;
+import com.clanki.ui.Ui;
+
+import java.util.ArrayList;
+import java.util.Objects;
+
+public class ReviewCommand extends Command {
+    private static void reviewCurrentFlashcard(Ui display, Flashcard flashcard) {
+        boolean isCurrentCardPassed = false;
```

Comment:
Since both branches of the correct/incorrect answers below end up assigning this variable to `true`, I'm a bit confused as in what this variable is for.

Reply:
When user gives a invalid input, it will stay false, so the flashcard will be shown again for user to review, instead of going to the next flashcard